sdk/objc/base/RTCAudioRenderer.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 LiveKit
```

Comment:
nit: new files should carry 2023

```suggestion
 * Copyright 2023 LiveKit
```